examples/step-61/doc/intro.dox

```diff
@@ -139,186 +139,333 @@ that is only defined in cell interiors. Consequently,
+that corresponds to the velocity with which the medium flows in a porous
+medium if this is the model we are trying to solve. This kind of
+step -- computing a derived quantity from the solution of the discrete
+problem -- is typically called "post-processing".
 
-On each element the gradient of the numerical pressure $\nabla p$ can be
-approximated by discrete weak gradients  $ \nabla_{w,d}\phi_i$, so
+Here, instead of using the exact gradient of $p_h$, let us instead
+use the discrete weak gradient of $p_h$ to calculate the velocity on each element.
+As discussed above,
+on each element the gradient of the numerical pressure $\nabla p$ can be
+approximated by discrete weak gradients  $ \nabla_{w,d}\phi_i$:
 @f{equation*}
-\nabla_{w,d} p_h = \sum_{i} a_i \nabla_{w,d}\phi_i.
+\nabla_{w,d} p_h 
+= \nabla_{w,d} \left(\sum_{i} P_i \phi_i\right) 
+= \sum_{i} P_i \nabla_{w,d}\phi_i.
 @f}
 
-The numerical velocity $ \mathbf{u}_h = -\mathbf{K} \nabla_{w,d}p_h$ can be written as
-@f{equation*}
-\mathbf{u}_h = -\mathbf{K} \nabla_{w,d} p = 
--\sum_{i} \sum_{j} a_ic_{ij}\mathbf{K}\mathbf{w}_j,
+On cell $K$,
+the numerical velocity $ \mathbf{u}_h = -\mathbf{K} \nabla_{w,d}p_h$ can be written as
+@f{align*}{
+\mathbf{u}_h 
+&= -\mathbf{K} \nabla_{w,d} p = 
+-\mathbf{K}\sum_{i} \sum_{j} P_i C^K_{ij}\mathbf{v}_j
+&= -\nabla_{w,d} p = 
+-\mathbf{K}\sum_{i} \sum_{j} P_i C^K_{ij}\mathbf{K} \mathbf{v}_j,
 @f}
-where $c_{ij}$ is the coefficient of Gram matrix, 
-$\mathbf{w}_j$ is the basis function of the $RT$ space. 
-$\mathbf{K} \mathbf{w}_j$ may not be in the $RT$ space. 
-So we need $L_2$-projection to project it back to the $RT$ space. 
+where $C^K$ is the expansion matrix from above, and
+$\mathbf{v}_j$ is the basis function of the $RT$ space on a cell.
+
+Unfortunately, $\mathbf{K} \mathbf{v}_j$ may not be in the $RT$ space. 
+So, in order to represent it in a finite element program, we need to
+project it back into a finite dimensional space we can work with. Here,
+we we will use the $L_2$-projection to project it back to the (broken) $RT$
+space. 
 
 We define the projection as 
-$ \mathbf{Q}_h \left( \mathbf{K}\mathbf{w}_j \right) = 
-\sum_{k} d_{jk}\mathbf{w}_k$. 
+$ \mathbf{Q}_h \left( \mathbf{K}\mathbf{v}_j \right) = 
+\sum_{k} d_{jk}\mathbf{v}_k$ on each cell $K$. 
 For any $j$, 
-$\left( \mathbf{Q}_h \left( \mathbf{Kw}_j \right),\mathbf{w}_k \right)_E = 
-\left( \mathbf{Kw}_j,\mathbf{w}_k \right)_E.$ 
-So the numerical velocity becomes
+$\left( \mathbf{Q}_h \left( \mathbf{Kv}_j \right),\mathbf{v}_k \right)_K = 
+\left( \mathbf{Kv}_j,\mathbf{v}_k \right)_K.$ 
+So, rather than the formula shown above, the numerical velocity on cell $K$
+instead becomes
 @f{equation*}
 \mathbf{u}_h = \mathbf{Q}_h \left( -\mathbf{K}\nabla_{w,d}p_h \right) = 
--\sum_{i=0}^{4} \sum_{j=1}^{4}a_ib_{ij}\mathbf{Q}_h \left( \mathbf{K}\mathbf{w}_j \right),
+-\sum_i \sum_j P_i B^K_{ij}\mathbf{Q}_h \left( \mathbf{K}\mathbf{w}_j \right),
```

Comment:
```suggestion
-\sum_i \sum_j P_i B^K_{ij}\mathbf{Q}_h \left( \mathbf{K}\mathbf{v}_j \right),
```
(I think you meant to convert all *w*s to *v*s)

Reply:
Correct. Thanks for paying attention!